EIPS/eip-4911.md

```diff
@@ -0,0 +1,157 @@
+---
+eip: 4911
```

Comment:
```suggestion
eip: xxxx
```
An EIP number will be assigned when you move this PR out of draft. Unfortunately you can't pick a specific EIP number.